website/docs/getting-started/setup/instance-configuration/custom-domain/README.md

```diff
@@ -1,95 +1,75 @@
 ---
 sidebar_position: 7
 ---
-# SSL & Custom Domain
+# SSL for a Custom Domain
 
-You can make Appsmith available on a custom domain by updating your domain's DNS records to point to the instance running Appsmith. Most domain registrars / DNS providers have documentation on how you can do this yourself.
+You can configure SSL for your Custom Domain to establish secured connections. This page provides instructions to set up SSL for your Custom Domain on your Appsmith instance.
```

Comment:
```suggestion
This page provides instructions to configure SSL for your custom domain to establish secured connections on your Appsmith instance.
```

Reply:
This sentence becomes too difficult to read. Will reword to:
`This page explains how to set up SSL for your custom domain on the Appsmith instance. So that you can make secure connections.`